package.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "mine.js",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "node.JS implementation of a OpenMined data mine",
-  "main": "app.js",
+  "main": "lib/mines.js",
   "private": true,
   "bin": {
-    "mine": "cli/mine.js"
+    "mine": "bin/cli.js"
   },
   "scripts": {
     "test": "standard && nyc ava",
     "test:coverage": "nyc ava",
     "test:watch": "ava --watch",
     "env-start": "docker-compose -f setup/docker-compose.yml up -d",
     "env-stop": "docker-compose -f setup/docker-compose.yml stop",
-    "start": "nodemon app",
+    "start": "nodemon ./bin/cli.js --mine-address 0xb8ce9ab6943e0eced004cde8e3bbed6568b2fa01 --contract-address 0xb8ce9ab6943e0eced004cde8e3bbed6568b2fa01",
```

Comment:
oooooh much better. 

My end goal would be to just add a --dev flag to add watching, but it looks like we'd have to move to something other than `nodemon` for that.